microsoft-edge/devtools-guide-chromium/device-mode/index.md

```diff
@@ -58,7 +58,7 @@ By default the Device Toolbar opens in Responsive Viewport Mode.
 
 To quickly test the look and feel of your page across multiple screen sizes, drag the handles to resize the viewport to your required dimensions.  You can enter any numeric values in the width and height boxes.  If you select a size larger than available in the browser window, the viewport will be automatically scaled to accommodate for the larger viewport.
 
-In the following figure, the width is set to `626` and the height is set to `516`.
+In the following figure, the width is set to `400` and the height is set to `736`.
 
 ![The handles for changing the dimensions of the viewport when in Responsive Viewport Mode](./index-images/device-mode-toggle-device-toolbar-handles-highlighted.png)
 
```

Comment:
not really about updating the .pngs, this comment is about the need to use Action > Result format.  
Rendered section:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/device-mode/?branch=pr-en-us-2695#show-media-queries
I was confused, it wasn't immediately clear, what exactly the breakpoints UI is.   Reworking the article or article section as standard, consistent Action > Result format would double-describe the UX, both as text and supporting .pngs.  I'm focusing here on the colored bands that represent breakpoints (browser width) - they ought to be mentioned in text and not just shown in a png with a red box.

Reply:
Can we address this in a follow-up PR? I'm trying to put the finishing touches to the screenshot updates and would prefer to handle this at a later stage.

Reply:
@captainbrosset - I linked to the present comment from a new comment in my PR
https://github.com/MicrosoftDocs/edge-developer/pull/2726/files#r1272557176, to be dealt with in that PR.

Reply:
can Resolve thread.  Fixed in the other PR; made article use more standard format.